packages/launcher/src/main/db/collections/users.js

```diff
@@ -127,6 +127,12 @@ export default async (params: CollectionParams) => {
         await this.update({ $push: { 'ethWallets.ledger': id } })
       },
 
+      async setProfileEthAddress(address: string) {
+        const profile = { ...this.profile }
+        profile.ethAddress = address
+        await this.update({ $set: { profile } })
```

Comment:
Not an issue but I think this could be simply set as `await this.atomicSet('profile.ethAddress', address)`